src/Misc/layoutbin/en-US/strings.json

```diff
@@ -254,6 +254,7 @@
   "RMDownloadingJenkinsArtifacts": "Downloading the artifacts from Jenkins server",
   "RMEnvironmentVariablesAvailable": "Environment variables available are below.  Note that these environment variables can be referred to in the task (in the ReleaseDefinition) by replacing \"_\" with \".\" e.g. AGENT_NAME environment variable can be referenced using Agent.Name in the ReleaseDefinition: {0}",
   "RMErrorDuringArtifactDownload": "An error occurred during download: {0}",
+  "RMGitEndpointNotFound": "Cannot find required information in the job to download the Team Foundation Git artifact.",
```

Comment:
Its not required. Once we split the records for each artifact then its straight forward to find which artifact it is. including the name